src/prefect/cli/execute.py

```diff
@@ -0,0 +1,72 @@
+import click
+
+import prefect
+from prefect.client import Client
+from prefect.utilities.graphql import with_args
+
+
+@click.group(hidden=True)
+def execute():
+    """
+    Execute flow environments.
+
+    \b
+    Usage:
+        $ prefect execute [OBJECT]
+
+    \b
+    Arguments:
+        local-flow  Execute a local flow's environment
+        cloud-flow  Execute a cloud flow's environment (during deployment)
+
+    \b
+    Examples:
+        $ prefect execute cloud-flow
+
+    \b
+        $ prefect execute local-flow ~/.prefect/flows/my_flow.prefect
+    """
+    pass
+
+
+@execute.command(hidden=True)
+def local_flow():
+    """
+    Execute a flow's environment locally.
+    """
+    pass
```

Comment:
If we aren't going to implement this in this PR (which is fine), I vote we remove it for now